tests/integration/assetmanagement/test_asset_management.py

```diff
@@ -221,27 +180,27 @@ def test_query_assets_with_projections__returns_the_assets_with_projected_proper
             and asset.custom_calibration_interval is None
             and asset.discovery_type is None
             and asset.external_calibration is None
-            and asset.file_ids is None
+            and asset.file_ids == []
```

Comment:
When you're using a projection these should be None not empty lists.

Reply:
That sounds like a bug in the service's projection implementation. @CiprianAnton is this the behavior you expect?